src/models/Preset.ts

```diff
@@ -7,7 +7,7 @@ class Preset {
 
     public static readonly EMPTY: Preset = new Preset('', [], []);
 
-    public static readonly NEW: Preset = new Preset('Change me', Civilisation.ALL, []);
+    public static readonly NEW: Preset = new Preset('', Civilisation.ALL, []);
```

Comment:
I would like to have some text in there by default. It does not necessarily have to be "Change me".

Reply:
I had added placeholder in the input box to take care of that. Is that sufficient?
![image](https://user-images.githubusercontent.com/13775/84584212-0dfb1780-ae1f-11ea-9ab7-62b57edad62c.png)

Reply:
No, because if you ignore the input box and click on the "create" button straight away, as it often happens, you get a draft without title that shows up in the latest drafts list like that as well.

Reply:
I guess it's either we 
- leave it like that, resulting sometimes in empty titles,
- make that a mandatory field,
- pre-fill it with a fixed value like "change me", "Noob Draft", …
- pre-fill it with a random value like "{adjective} Draft", where `adjective` comes from a list of "Amazing", "Adorable", "Marvellous", "Magnificient", …

Reply:
Got it. I have reverted this and the current behaviour is:
> pre-fill it with a fixed value like "change me", "Noob Draft", …

We can address the validation here and also in "Enter code to Join Draft / Spectate" screens and "Chose name" modal in the subsequent PRs.